kuwala/pipelines/google-poi/resources/categories.json

```diff
@@ -81,24 +107,48 @@
         "category": "education",
         "tags": [
             "library",
+            "college",
+            "public_university",
             "public_library",
-            "school"
+            "school",
+            "english_language_school",
+            "language_school",
+            "nursery_school",
+            "adult_education_school",
+            "private_school",
+            "school_supply_store",
+            "after_school_program",
+            "music_school",
+            "trade_school",
+            "religious_school",
+            "ballet_school",
```

Comment:
I guess this fits better in "fitness", no?

Reply:
Ah, the category is "sport" not "fitness". Maybe there is a better name for the recreation. It's mainly park, lakes, beaches and not 100% sport related. Is leisure fitting better? What do you think?

Reply:
I think having a leisure category would be too broad. It would suck up a fair amount of tags from the recreation category. I believe it would be suited to the sport category

Reply:
Okay, then let's keep it that way